src/routes/NavBar.svelte

```diff
@@ -37,7 +37,7 @@
       {#if $page.data.session}
         <button class="btn btn-neutral" on:click={() => signOut()}> Logga ut </button>
         {#if user?.student_id}
-          <a class="btn btn-ghost" href="/member/{user.student_id}">
+          <a class="btn btn-ghost" href="/members/{user.student_id}">
```

Comment:
Did we revert the change to `member`?

Reply:
Yes, because the old site uses /members/ and it's hard-coded into for example comments, and I could guess for example news articles